docs/map.xsd

```diff
@@ -17,24 +17,89 @@
   types
 -->
 
+  <xs:simpleType name="pixelT">
+    <xs:restriction base="xs:nonNegativeInteger"></xs:restriction>
+  </xs:simpleType>
+
 <xs:simpleType name="encodingT">
   <xs:restriction base="xs:NMTOKEN">
     <xs:enumeration value="base64" />
+    <xs:enumeration value="csv" />
   </xs:restriction>
 </xs:simpleType>
 
 <xs:simpleType name="compressionT">
   <xs:restriction base="xs:NMTOKEN">
     <xs:enumeration value="gzip" />
+    <xs:enumeration value="zlip" />
   </xs:restriction>
 </xs:simpleType>
-
+  
 <xs:simpleType name="nameT">
   <xs:restriction base="xs:string">
     <xs:minLength value="1"/>
   </xs:restriction>
 </xs:simpleType>
 
+  <xs:simpleType name="draworderT">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="index" />
+      <xs:enumeration value="topdown" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="staggeraxisT">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="x" />
+      <xs:enumeration value="y" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="imageformatT">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="png" />
+      <xs:enumeration value="gif" />
+      <xs:enumeration value="jpg" />
+      <xs:enumeration value="bmp" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="degreesT">
+    <xs:restriction base="xs:nonNegativeInteger">
+      <xs:maxInclusive value="360"/>
+      <xs:minInclusive value="0"/>
```

Comment:
I just noticed this. There's nothing requiring the degrees to be normalized between 0 and 360. Also, they aren't required to be integer.